apps/server/src/shared/repo/task/task.repo.ts

```diff
@@ -94,7 +94,11 @@ export class TaskRepo {
 			courseIds?: EntityId[];
 			lessonIds?: EntityId[];
 		},
-		filters?: { draft?: boolean; afterDueDateOrNone?: Date; finished?: { userId: EntityId; value: boolean } },
+		filters?: {
+			afterDueDateOrNone?: Date;
+			finished?: { userId: EntityId; value: boolean };
+			availableOn?: Date;
+		},
```

Comment:
Why filters Prop is not a Type or DTO?

Reply:
yes. absolutely. but we have to look into that later.